app/upgrades/v8/README.md

```diff
@@ -0,0 +1,10 @@
+# Upgrade v8 Changelog
+1. v8 Upgrade Handler (incl. adding evmos airdrop and reseting existing airdrops) ([]())
```

Comment:
We've been only including the on chain changes in the upgrade directory. 

Also same comment here with 1 and 4 - I've considered them on chain